pyproject.toml

```diff
@@ -106,3 +106,10 @@ initial-content = """
   __version__ = "0.0.0"
   __version_tuple__ = (0, 0, 0)
 """
+[tool.numpydoc_validation]
+checks = ["all", #report on all checks, except the following
+    "EX01", # Ignore No examples section found
+    "SA01", # Ignore See Also section not found
+    "ES01" ] # Ignore No extended summary found
```

Comment:
As we discussed, you can also add "RT02" to this list ("The first line of the Returns section should contain only the type, unless multiple values are being returned")